src/index.tsx

```diff
@@ -15,35 +16,17 @@ import './css/common/pp-semantic-ui-reset.scss';
 import './css/common/pp-semantic-ui-overrides.scss';
 
 import './css/selection.scss';
-import {showEditorNewAnnotation} from 'store/widgets/actions';
-import {getAnnotationUrl} from './utils/url'
 
 console.log('Przypis script working!');
 
-function injectApp() {
-  const documentContainer = document.createElement('div');
-  documentContainer.id = 'pp-document-container';
-  window.document.body.appendChild(documentContainer);
-
-  // TODO: 1. wrap those dispatches into dedicated function to make this code more separate and descriptive
-  // TODO: 2. this url is hardcoded now, but should be imported from config
-  store.dispatch(setAxiosConfig({
-    baseURL: 'http://localhost:8000/api',
-  }));
-
-  // This our root request that need to have part of the url (path) hardcoded
+function loadData() {
+  // This is our root request that needs to have part of the url (path) hardcoded
   store.dispatch(readEndpoint('/annotations?url=' + getAnnotationUrl()));
-
```

Comment:
`getAnnotationUrl` can be safely replaced with normal,raw and universal `window.location` thanks to changes from incoming https://github.com/PrzypisPowszechny/pp/pull/16 which will handle full urls. Both when creating annotation and then when listing annotation with filtering query (`?url=` ). The original, full url will be stored as well as processed version used for efficient and accurate queries, with the results based on the relevant part of url only (without protocol etc).

Reply:
I'll do it right away